tests/test_real_data_suggestions.py

```
@@ -0,0 +1,302 @@
+"""Real-data driven tests for the SuggestionEngine and ParameterOptimizer.
+
+These tests load cycle exports from cycle_data/ (including user-contributed data)
+and verify that the suggestion pipeline produces sensible parameter recommendations.
+They are not strict unit tests – they validate *plausibility* bounds so that
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)